src/FizzWare.NBuilder/Generators/GetRandom.cs

```diff
@@ -345,9 +345,11 @@ public static Guid Guid()
             return generator.Guid();
         }
 
+
         /// <summary>
         /// Generate a random first name from a pre-defined list of names.
         /// </summary>
+        [Obsolete("Use new ValueSet<string>(name1, name2...) instead.")]

```

Comment:
I thought `FirstName`, `LastName` and `Phrase` were not going to be touched and the ValueSet was mainly about providing a collection of data for _specific_ data, like cities or postcodes or some custom list of <somethings>.

I would have thought that these three methods would be used _heaps_ and the idea here is that we (the developer/consumer) doesn't have to think up of any names and just use the generic ones?

Reply:
I'd like to standardize on an approach. I'm open to creating a couple of predefined ValueSets for convenience if there's still value in having them.

Reply:
I think my reply [here](https://github.com/nbuilder/nbuilder/issues/104#issuecomment-660529520) seems relevant to this particular conversation. Having some out-of-the-box support is great, but let the consumer specify their own set of options if they want.